netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -243,6 +251,10 @@ private static Provider findJdkProvider() {
             || JettyTlsUtil.isJava9AlpnAvailable()) {
           return provider;
         }
+      } else if (IBM_PROVIDER_NAME.equals(provider.getName())) {
```

Comment:
Because of the insertion of this else clause, the behavior is now different when using IBM JRE and both Conscrypt and IBMJSSE2 are present. It will now pick IBMJSSE2 over Conscrypt. If we want to keep the old behavior the else clause will need to be the last one.

Reply:
Security.getProviders() returns them in preference order. This if/else just checks whether the current provider of the loop is understood to support ALPN.